lib/Entity/DataSet.php

```diff
@@ -504,6 +504,7 @@ public function getData($filterBy = [], $options = [])
 
         // Are there any client side formulas?
         if (count($clientSideFormula) > 0) {
+            $language = '';
```

Comment:
any reason this is declared above? could it all be one statement inside the `if (isset($details[2])) {` check?